workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationInputs/isAuthorizedOnRoute.ts

```diff
@@ -3,30 +3,31 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
-import { AuthenticatedUser } from '../../authenticatedUser';
-import { HTTPMethod } from '../../routesMap';
+import { z } from 'zod';
+import { AuthenticatedUserParser } from '../../authenticatedUser';
+import { HTTPMethodParser } from '../../routesMap';
 
-/**
- * Request object for IsAuthorizedOnRoute
- */
-export interface IsAuthorizedOnRouteRequest {
+// eslint-disable-next-line @rushstack/typedef-var
+export const IsAuthorizedOnRouteRequestParser = z.object({
   /**
    * {@link AuthenticatedUser}
    */
-  authenticatedUser: AuthenticatedUser;
-
+  authenticatedUser: AuthenticatedUserParser,
   /**
    * The route the user is requesting access to
    */
-  route: string;
-
+  route: z.string(),
   /**
    * {@link HTTPMethod}
    */
-  method: HTTPMethod;
-
+  method: HTTPMethodParser,
   /**
    * Optional params for variable based operations
    */
-  params?: Record<string, unknown>;
-}
+  params: z.record(z.string()).optional()
```

Comment:
Is there a reason this was reverted back to `z.record(z.string()).optional()` from `z.record(z.unknown()).optional()`? The current change doesnt match the original `Record<string, unknown>`

Reply:
yeah, users are only capable of providing strings. Unknown isn't necessary